ansible_collections/f5networks/f5_modules/plugins/modules/bigip_device_license.py

```diff
@@ -49,11 +49,13 @@
       - The state of the license on the system.
       - When C(present), only guarantees that a license is there.
       - When C(absent), removes the license on the system.
+      - When C(latest), ensures that the license is always valid. This is not idempotent state since re-run can modify result
```

Comment:
fullstop missing

Reply:
I have added fullstop. This comment is addressed